### DISCUSSION TITLE
How to use timm encoder with Unet

### DISCUSSION THREAD

**Discussion by Athrun9z** [qubvel/segmentation_models.pytorch#854] [Q&A]
I saw a way to use smp on kaggle, the author of the scenario used unet from smp but added encoder and decoder from the timm library in the process instead of using several backbones that are already provided, I'm very curious about the way to do this, can someone tell me how it's done?

Here is the link to the solution: https://github.com/jing1tian/blood-vessel-segmentation
you can see the change about the segmentation_models_pytorch there

**Comment by JulienMaille** [comment]
Why don't you ask @jing1tian?

**Reply by Athrun9z** [reply]
Yes, I asked him on kaggle, but he hasn't answered me yet, maybe I'm too late, I saw there was a discussion about timm backbone here, so I wondered if I could find the answer here.
